eventhubs/data-plane/azure-eventhubs-eph/src/main/java/com/microsoft/azure/eventprocessorhost/PartitionManagerOptions.java

```diff
@@ -13,29 +13,29 @@
     /**
      * The default duration after which a partition lease will expire unless renewed.
      */
-    public final static int DefaultLeaseDurationInSeconds = 30;
+    public static final int DEFAULT_LEASE_DURATION_IN_SECONDS = 30;
```

Comment:
We can't rename public static fields - these form part of the public API. You should chat with @conniey about how she handled this by adding suppressions.